client/src/components/graph/graph.tsx

```diff
@@ -542,7 +549,18 @@ class Graph extends React.Component<GraphProps, GraphState> {
   };
 
   getViewportDimensions = (): { height: number; width: number } => {
-    const { viewportRef } = this.props;
+    const { viewportRef, screenCap } = this.props;
+
+    if (screenCap) {
+      const prevAspectRatio =
+        viewportRef.clientHeight / viewportRef.clientWidth;
+
+      // seve: Default to 1080p resolution (arbitrary, but a good starting point for screen captures)
+      return {
+        height: 1080 * prevAspectRatio,
+        width: 1080,
```

Comment:
Possible to dynamically determine the user's screen resolution like this, using the `screen` object, instead of hardcoding it?

```
const screenWidth = screen.width;
const screenHeight = screen.height;
```

Reply:
we aren't programatically determining the user's screeen size. We're grabbing the aspect ratio of the viewport which the graph is rendered, and maintaining that aspect ratio while setting the width to a constant size.